frontend/src/__tests__/cypress/cypress/pages/pipelines/experiments.ts

```diff
@@ -30,29 +30,57 @@ class ExperimentsTabs {
     return new ExperimentsTable(() => cy.findByTestId('experiments-archived-tab-content'));
   }
 
-  mockGetExperiments(
-    namespace: string,
-    activeExperiments: ExperimentKFv2[],
-    archivedExperiments: ExperimentKFv2[] = [],
-  ) {
+  mockGetExperiments(namespace: string, experiments: ExperimentKFv2[]): Cypress.Chainable<null> {
     return cy.interceptOdh(
       'GET /api/service/pipelines/:namespace/:serviceName/apis/v2beta1/experiments',
       {
         path: { namespace, serviceName: 'dspa' },
       },
       (req) => {
-        const { predicates } = JSON.parse(req.query.filter.toString());
-
-        if (predicates.length === 0) {
-          req.reply({ experiments: activeExperiments, total_size: activeExperiments.length });
-        } else {
-          const [{ string_value: experimentState }] = predicates;
-          if (experimentState === 'ARCHIVED') {
-            req.reply({ experiments: archivedExperiments, total_size: archivedExperiments.length });
-          } else {
-            req.reply({ experiments: activeExperiments, total_size: activeExperiments.length });
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        const { filter, sort_by, page_size } = req.query;
+        let results = experiments;
+        if (sort_by) {
+          const fields = sort_by.toString().split(' ');
+          const sortField = fields[0];
+          const sortDirection = fields[1];
+          // more fields to be added
```

Comment:
What fields still need to be added?

Reply:
Like `name`, or any other sortable fields. I don't want to spend much time adding/updating test cases in this PR so I only updated the related fields.